src/components/guidance-content/about/ClimateExplorer.js

```diff
@@ -17,11 +17,11 @@ export default function ClimateExplorer() {
             <ListGroupItem header='Description'>
               <p>
                 A tool for visualizing and downloading
-                climate model data and data derived from climate outputs
-                for the B.C. - Yukon region.
+                climate model outputs and data derived from those outputs.
               </p>
               <p>
-                Climate Explorer is also fondly known as 'the Marmot,'
+                The PCIC Climate Explorer (PCEX) is also fondly known as
+                the 'marmot,'
```

Comment:
An unrequested (though trivial) change you've made:

`'the marmot'` 
to 
`the 'marmot'`

Reply:
It was intentional; it's one of those trad vs. modern things. Trad says punctuation goes inside quotes. I'm kind of a trad guy, but I really don't care very much.

Reply:
I  certainly agree with you on that! The difference I was commenting on is that "the" was previously inside the quotes and is now outside them.